x-pack/docs/en/rest-api/security/oidc-logout-api.asciidoc

```diff
@@ -48,7 +48,7 @@ POST /_security/oidc/logout
   "refresh_token": "vLBPvmAB6KvwvJZr27cS"
 }
 --------------------------------------------------
-// TEST[catch:unauthorized]
+// TEST[catch:request]
```

Comment:
The (expected for this test) 500 was masked by the `401` that was thrown in https://github.com/elastic/elasticsearch/blob/d029a131ba116927a8b005533cb56f1a24e0d9a8/x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/action/oidc/TransportOpenIdConnectLogoutAction.java#L55

Reply:
this is necessary because we don't do the login flow so we don't have tokens to invalidate for the logout. We expect that this call would fail for that reason - but it failing with a `401` was a mistake